### ISSUE TITLE
Rendering bug

### ISSUE THREAD

**Issue by NickHu** [homotopy-io/homotopy-rs#1183]
There seems to be some kind of problem with the rendering at the moment. I made the swallowtail (see attached), and it should be a monochromatic (yellow) picture; the blue part is erroneous.
![image](https://github.com/homotopy-io/homotopy-rs/assets/450276/009eac9f-310a-4fa7-89fc-3d7378c59168)
[swallowtail.zip](https://github.com/homotopy-io/homotopy-rs/files/12242994/swallowtail.zip)

**Comment by calintat** [comment]
I disagree. If you look at the slices of the diagram, they all have as their target the identity on blue so the max dimensional generator is blue. Also if you look at the diagram in 3D, you only see the yellow bit, but if you imagine the background is blue, then you get exactly the projected 2D picture.
